### PR TITLE
Move to Openssl 1.1.1f

### PR DESCRIPTION
**Pull request by SalusaSecondus**
*Description of changes:*

Move to Openssl 1.1.1f

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,7 +27,7 @@
 ## 1.2.0
 
 ### Improvements
-* Now uses [OpenSSL 1.1.1.d](https://www.openssl.org/source/openssl-1.1.1d.tar.gz). [PR #60](https://github.com/corretto/amazon-corretto-crypto-provider/pull/60)
+* Now uses [OpenSSL 1.1.1d](https://www.openssl.org/source/openssl-1.1.1d.tar.gz). [PR #60](https://github.com/corretto/amazon-corretto-crypto-provider/pull/60)
```
Comment (AzeemJiva) [inline CHANGELOG.md:30]: Should be 1.1.1f?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Amazon Corretto Crypto Provider
-The Amazon Corretto Crypto Provider (ACCP) is a collection of high-performance cryptographic implementations exposed via the standard [JCA/JCE](https://docs.oracle.com/en/java/javase/11/security/java-cryptography-architecture-jca-reference-guide.html) interfaces. This means that it can be used as a drop in replacement for many different Java applications. Currently algorithms are primarily backed by OpenSSL's implementations (1.1.1d) but this may change in the future.
+The Amazon Corretto Crypto Provider (ACCP) is a collection of high-performance cryptographic implementations exposed via the standard [JCA/JCE](https://docs.oracle.com/en/java/javase/11/security/java-cryptography-architecture-jca-reference-guide.html) interfaces.
+This means that it can be used as a drop in replacement for many different Java applications.
```
Comment (AzeemJiva) [inline README.md:3]: Suggested rewording:

* ACCP can be used as a drop-in replacement for many Java applications

File: README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
+Currently algorithms are primarily backed by OpenSSL's implementations (1.1.1f as of ACCP 1.4.0) but this may change in the future.
```
Comment (AzeemJiva) [inline README.md:4]: * The cryptographic algorithms are backed by OpenSSL 1.1.1f as of ACCP 1.4.0, but this is subject to change.

File: README.md (hunk 2 above)
Comment (SalusaSecondus) [reply README.md:3]: I'll look into cleaning up the readme in a separate PR.

File: README.md (hunk 3 above)
Comment (SalusaSecondus) [reply README.md:4]: We implement HMAC and AES-CTR-DRBG ourselves (though backed by openssl primitives). So, while this language is a bit longer, I feel it is more accurate, and so will keep it.

**Comment by SalusaSecondus** [comment]
`accp_github_pr` failed due to the known testing heap space issue. Restarting the test and will close/open the PR if needed to restart the flow.

**Comment by SalusaSecondus** [comment]
`accp_github_pr` failed due to the known testing heap space issue. Restarting the test and will close/open the PR if needed to restart the flow.

**Pull request merged**